src/main/java/com/reimbursements/reporitories/ReimbursementRepository.java

```diff
@@ -0,0 +1,17 @@
+package com.reimbursements.reporitories;
+
+import java.util.List;
+import java.util.Optional;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+import com.reimbursements.models.Reimbursement;
+import com.reimbursements.models.User;
+
+
+public interface ReimbursementRepository extends JpaRepository<Reimbursement, Integer>{
+
+	
+	public Optional<List<Reimbursement>> getByAuthor(User author);

```

Comment:
You don't need the `Optional` here. By default, when a list is fetched, spring won't throw an exception if there's no result, but an empty list will be returned. There always will be an object returned, so `Optional` is useless here.